install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#TODO(by black7375)
+#However, there may be existing user's settings, so back up and make a copy.
+#I just made a simple one yesterday, assuming that there may be multiple backup states.
+#I think it's better to do a redirection(>>) after backup to merge the contents of the file.
+printf "Are you using ESR, Default, or Dev?(ESR/Default/Dev): "
+read -r
+if [ "ESR" == "$REPLY" ] || [ "$REPLY" == "esr" ]; then
+  profiledir="$(grep '.default-esr' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "Default" == "$REPLY" ] || [ "default" == "$REPLY" ]; then
+  profiledir="$(grep '.default-release' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "Dev" == "$REPLY" ] || [ "dev" == "$REPLY" ]; then
+  profiledir="$(grep '.dev-edition-default' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+else
+  echo "Unspecified."
+  exit
+fi
+printf "Do you want to disable everything being selected when you click at the URL bar?(N/y): "
+read -r
+if [ "y" == "$REPLY" ] || [ "Y" == "$REPLY" ]; then
+  curl -s -L https://raw.githubusercontent.com/7k5x/firefox-selection-fix/master/fixfx-selection.sh | bash
```

Comment:
We need to fix the `firefox-selection-fix` code.
Refer: https://github.com/black7375/Firefox-UI-Fix/issues/32#issuecomment-853096949

Reply:
Do you mean I should cut off the whole asking for select part?

Reply:
@black7375 should I delete the asking+installing selectfix part?

Reply:
No, changing the defaults requires porting to us.
Let's deal with it in the next PR.

Reply:
> @black7375 should I delete the asking+installing selectfix part?

yes.